Source/Plugins/Core/com.equella.core/resources/web/sass/legacy.scss

```
@@ -5197,3 +5199,22 @@ label[for="password"] {
 button:focus {
   outline: lightgrey dashed 1px;
 }
+
+.manualdatafixes {
+  button {
+    @include buttonShadowBorder;
+    margin-top: $singleMargin;
+    margin-bottom: $singleMargin;
+  }
+  #rmdfsgto_forceUpdate {
```

Wowsers! That's some ID hey? Obviously generated, so probably best to add a comment here to say what this is for. Is it something like:

```suggestion
  // The radio buttons for the 'Generate thumbnails and previews' section
  #rmdfsgto_forceUpdate {
```